app/src/main/java/de/danoeh/antennapod/fragment/preferences/PlaybackStatisticsFragment.java

```diff
@@ -79,7 +79,9 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
     @Override
     public void onStart() {
         super.onStart();
-        ((PreferenceActivity) getActivity()).getSupportActionBar().setTitle(R.string.statistics_label);
+        if (getActivity().getClass() == PreferenceActivity.class) {
+            ((PreferenceActivity) getActivity()).getSupportActionBar().setTitle(R.string.statistics_label);
+        }
```

Comment:
Oh, I think those lines can all just be removed. The title in settings is already handled by `StatisticsFragment` (where you added a similar change). That's a leftover from the old statistics screen with only one single tab.